sdk/azurestack/Microsoft.Azure.Management.AzureStack/Registration.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
+VisualStudioVersion = 15.0.28307.645
+MinimumVisualStudioVersion = 10.0.40219.1
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{0A440EAF-0500-476E-83EE-C3AF80AB253A}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Azure.Management.AzureStack", "src\Microsoft.Azure.Management.AzureStack.csproj", "{1EC06268-BF25-4860-8DE7-D1A5CD6AF980}"
```

Comment:
I would prefer this to be called as Microsoft.Azure.Management.AzureStack.Registration @sarathys  wt do you think?

Reply:
I have no problem with changing the .csproj name, but in the code the namespace is _Microsoft.Azure.Management.AzureStack_, which is auto-generated.